### PR TITLE
Enhanced metadata and /media endpoint

### PR DESCRIPTION
**Pull request by Julusian**
This adds some /media endpoints which return json information on the media. Defaults to disabled as it is only useful to those who will connect to the scanner directly.

This also sets the content-types of the other endpoints to text/plain.
And fixes a bug from #19 where nothing stale would be deleted from the last page

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -23,6 +23,11 @@ const defaults = {
     width: 256,
     height: -1
   },
+  metadata: {
```
Comment (ronag) [inline src/config.js]: skip the enhanced true/false, Just check if `metadata != null`.

File: src/scanner.js [hunk 2]
```diff
@@ -194,28 +195,124 @@ module.exports = function ({ config, db, logger }) {
...
+        '-an',
+        '-f', 'rawvideo', '-y', (process.platform === 'win32' ? 'NUL' : '/dev/null'),
+        '-i', `"${doc.mediaPath}"`
+      ]
+      cp.exec(args.join(' '), (err, stdout, stderr) => {
+        if (err) {
+          return reject(err)
+        }
+
+        var regex2 = /Multi frame detection: TFF:\s+(\d+)\s+BFF:\s+(\d+)\s+Progressive:\s+(\d+)/
```
Comment (ronag) [inline src/scanner.js]: ´const`, better name?

File: src/config.js (hunk 1 above)
Comment (Julusian) [reply src/config.js]: Wouldn't that mean I cant define default settings here though, unless you want it to default to enabled?

Comment (ronag) [reply src/config.js]: default to enabled

**Comment by Julusian** [comment]
@ronag Both changes have been made

**Pull request merged**